apps/emqx_dashboard/src/emqx_dashboard.erl

```diff
@@ -247,10 +247,16 @@ authorize(Req) ->
                     {401, 'BAD_TOKEN', <<"Get a token by POST /login">>}
             end;
         _ ->
-            return_unauthorized(
-                <<"AUTHORIZATION_HEADER_ERROR">>,
-                <<"Support authorization: basic/bearer ">>
-            )
+            Path = cowboy_req:path(Req),
```

Comment:
```
authorize(Req) ->
    authorize(cowboy_req:path(Req), Req).

authorize(<<"/api/v5/status">>, _) ->
    ok;
authorize(_Path, Req) ->
    the old code block.
```

Reply:
this is maybe a better option:
https://github.com/emqx/emqx/blob/8fa9a0633739871a4835e8ce7226e2466848c378/apps/emqx_prometheus/src/emqx_prometheus_api.erl#L74